src/ONYXKEYS.ts

```diff
@@ -427,6 +427,15 @@ const ONYXKEYS = {
         // Shared NVPs
         /** Collection of objects where each object represents the owner of the workspace that is past due billing AND the user is a member of. */
         SHARED_NVP_PRIVATE_USER_BILLING_GRACE_PERIOD_END: 'sharedNVP_private_billingGracePeriodEnd_',
+
+        /** Expensify cards settings */
+        SHARED_NVP_PRIVATE_EXPENSIFY_CARD_SETTINGS: 'sharedNVP_private_expensifyCardSettings_',
+
+        /**
+         * Stores the card list for a given fundID and feed in the format: card_<fundID>_<bankName>
+         * So for example: card_12345_ExpensifyCard
```

Comment:
Sorry, apparently its with a space
```suggestion
         * So for example: card_12345_Expensify Card
```

Reply:
@allgandalf it's not the name of the card, it's the name of the card feed which is referred to as bankName internally. For this project, the feed / bankName is `Expensify Card` - we will be have more feeds introduced in the future.